idd/Energy+.idd.in

```diff
@@ -21069,10 +21073,570 @@ ComfortViewFactorAngles,
   A22, \field Surface 20 Name
        \type object-list
        \object-list AllHeatTranSurfNames
-  N20; \field Angle Factor 20
+  N20, \field Angle Factor 20
```

Comment:
So this could've been an actually extensible object instead of just making a really long list of fields.  Just add the right field tags to this IDD object, and add an entry in the schema generator code and it wouldn't have been this big object. 
 I'm mulling over what to do with this now.  Anyone else have strong opinions either way?  @mjwitte ?

Reply:
@Myoldmopar Well, yes, and no. The changes here have not made this object extensible (needs a `\begin-extensible` tag and as you mentioned a new list name in scripts/dev/generate_epJSON_schema/modify_schema.py). But even with that, we still make this big objects in the idd to support classic IDF Editor for some level of complexity.

Reply:
I hear you.  I'm not inclined to hold up a PR for this since we technically don't have a policy against it.  At some point in the future, presumably the point when we cut off IDF entirely, we will make a cleanup pass on the large objects looking for objects to simplify down.  For now this is OK, but not great.

Reply:
@Myoldmopar @mjwitte Oh, it seems that I clearly didn't know everything involved in making this extensible.  What would you like me to do with this?

Reply:
> At some point in the future, presumably the point when we cut off IDF entirely

When is that?